charts/posthog/templates/cronjob.yaml

```diff
@@ -0,0 +1,72 @@
+{{- if .Values.clickhouse.backup.enabled }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: clickhouse-backup-cron
+spec:
+  # every night
+  schedule: {{ .Values.clickhouse.backup.backup_schedule }}
+  concurrencyPolicy: "Forbid"
+  jobTemplate:
+    spec:
+      backoffLimit: 1
+      completions: 1
+      parallelism: 1
+      template:
+        spec:
+          restartPolicy: Never
+          containers:
+            - name: run-backup-cron
+              image: yandex/clickhouse-client:latest
+              imagePullPolicy: IfNotPresent
+              env:
```

Comment:
Same as above

Reply:
there is no point to have configurable this variable CLICKHOUSE_PORT, because current helm chart doesn't allow to update default CLICKHOUSE_PORT installation(pls, correct me if I'm wrong), therefore backup should have not configurable port parameter as well.

Reply:
Sure but I meant the whole snippet so that could be passed as map from `values.yaml`